test/methods/geom_relations.jl

```diff
@@ -331,3 +331,15 @@ end
         @test GO.crosses(GI.LineString([(-2.0, 2.0), (-4.0, 2.0)]), $poly7) == false
     end
 end
+
+@testset "Single-argument methods" begin
+    @test GO.contains(1) == Base.Fix2(GO.contains, 1)
```

Comment:
```suggestion
    @test GO.contains(geom1)(geom2) == GO.contains(geom1, geom2)
```


I think this is a better demonstration of what it does.